### DISCUSSION TITLE
retrieve image from mongodb and display it in nextjs

### DISCUSSION THREAD

**Discussion by rebazjabar** [vercel/next.js#44759] [Help]
Hello guys  im looking for the documentation or source how to store image in mongodb and display it from nextjs 
![maxresdefault](https://user-images.githubusercontent.com/32215251/211677233-1843491b-f94e-4040-b666-c40b6518efff.jpg)

**Comment by GabenGar** [comment]
You don't store images in a database.

**Reply by rebazjabar** [reply]
How could be??? What if i wannt store it??
